### DISCUSSION TITLE
Anthropogenic sounds

### DISCUSSION THREAD

**Discussion by GrunCrow** [birdnet-team/BirdNET-Analyzer#374] [Ideas]
I run BirdNET without an species list and it gave me a lot of anthropogenic sounds. When I selected LAT and LON it gave me a list without any anthropogenic sound. It would be great to have the possibility to check/uncheck if we want to add anthropogenic sounds in the species list. I've been looking in the general list and with a script to filter those classes where scientific name_common name was the same (I saw there was a pattern in that kind of sound classes) I got the following list:

Dog_Dog
Engine_Engine
Environmental_Environmental
Fireworks_Fireworks
Gryllus assimilis_Gryllus assimilis
Gun_Gun
Human non-vocal_Human non-vocal
Human vocal_Human vocal
Human whistle_Human whistle
Miogryllus saussurei_Miogryllus saussurei
Noise_Noise
Power tools_Power tools
Siren_Siren

I don't know if there are more anthropogenic sounds included in the general list. For the moment I just added those classes manually to the species list I have but I find it very useful for other users.

**Comment by fegue** [comment]
I believe you found all anthropogenic classes. At least, I am not aware of any other.  I wouldn't consider 'Gryllus assimilis' and 'Miogryllus saussurei' anthropogenic, though. They are insects.

If you have questions, I recommend consulting the R package I developed for BirdNET labels, which I mainly use internally. You can check it out here: https://github.com/OekoFor/Vogelnamen
